### DISCUSSION TITLE
Track changes on the online collaborative editor

### DISCUSSION THREAD

**Discussion by shashwatsridhar** [typst/typst#1499] [Ideas]
Hey, I just started using typst, and it looks really promising!

I wrote up a piece of text in the typst web app that I'd like to share with my colleagues, so they can recommend changes to the text. However, I couldn't find a "track changes" feature anywhere. 

I do think that would be a very nice addition to the collaborative editor to have a system to track changes.

Cheers,
Shashwat

**Comment by hagi123456** [comment]
I hope this is coming soon. This is the one thing that makes me still prefer overleaf over typst.app for collaborative writing. Just using comments is a bit cumbersome. 
But this feature is listed in the typst roadmap [1], so I hope it is coming soon. 

[1] https://typst.app/docs/roadmap/

**Comment by sverrirarnors** [comment]
Happy to bump this. I recently had a collaborative project that had to be ported over to Overleaf because of the lack of track changes.

**Reply by behinger** [reply]
This is the main feature missing for my lab as well (and acceptance of typst at publishers ;))

**Comment by mdaeron** [comment]
Is there a way to know the level of priority for "Change tracking" (as per the current roadmap) in the upcoming release(s)? For me that's the only real use-case for the web app: collaborative writing with an "executive" permission level needed to validate proposed edits. It would be a very substantial improvement for many Typst users. Just my 2c.

**Comment by svanterwingen** [comment]
I agree, this option should be added in the future.

**Comment by matthiaskoenig** [comment]
Yes, this is a major issue for migrating completely from overleaf. Especially for feedback on theses and manuscripts.
